CONTRIBUTING.md

```diff
@@ -4,45 +4,87 @@ Above all, thank you for your interest in both the Synthstrom Deluge and being a
+
+* Please submit an [issue](https://github.com/SynthstromAudible/DelugeFirmware/issues) highlighting as much *relevant* context about the bug as possible.
 * Quality of information (rather than quantity, necessarily) is key! See [How to Report Bugs Effectively](https://www.chiark.greenend.org.uk/~sgtatham/bugs.html) by Simon Tatham for more on this topic.
+* Do let us know whether the bug was found on an OLED or 7seg Deluge as this may be important as well as the source and version of firmware (branch/etc.).
+* Where the bug exists in build scripts, e2, or other development, building, and debugging processes, please provide excerpts of the relevant logs if you suspect they might be helpful.
+* Bugs in related projects should be submitted to the appropriate issue tracker:
+    * DelugeProbe: [litui/delugeprobe](https://github.com/litui/delugeprobe).
+    * DBT-Toolchain: [litui/dbt-toolchain](https://github.com/litui/dbt-toolchain).
 
-* Do let us know whether the bug was found on an OLED or 7seg Deluge as this may be important.
 
-* Where the bug exists in build scripts, e2, or other development, building, and debugging processes, please provide excerpts of the relevant logs if you suspect they might be helpful.
+# Ideas, feature requests, suggestions and discussions
 
-* Bugs in related projects should be submitted to the appropriate issue tracker:
-    * DelugeProbe: [litui/delugeprobe](https://github.com/litui/delugeprobe)
-    * DBT-Toolchain: [litui/dbt-toolchain](https://github.com/litui/dbt-toolchain)
+Please use the [Discussions](https://github.com/SynthstromAudible/DelugeFirmware/discussions) feature in our repository which allows both creating discussion, ideas, polls and more as well as voting on existing topics. 
+
+* Please search first if there is already a (draft) [Pull request](https://github.com/SynthstromAudible/DelugeFirmware/pulls) or existing Discussion item related to your idea and start your discussion from there if possible.
+* Reading, voting and commenting on discussion items and Pull requests will help a lot to improve the direction of the project.
+* Please don't use the repository [issue](https://github.com/SynthstromAudible/DelugeFirmware/issues) tracker only for bug reports.
+* For further inspiration and discussion there is also the [Deluge Suggestions area](https://forums.synthstrom.com/categories/deluge-suggestions) and [Open Source area](https://forums.synthstrom.com/categories/open-source-) on the [Synthstrom Forum](https://forums.synthstrom.com).
+* For more in-depth technical discussion you are welcome to join the #dev-chat channel in the [Deluge User Discord](https://discord.com/invite/gWaJEh6pPP).
+
+
+# Code review and testing
+
+Pull requests that are no longer marked as Draft and are open for merging should be ready to test and any help is appreciated. 
+
+* This includes both building and testing the firmware as well as reading and reviewing the source code.
```

Comment:
Some verbiage about bare minimum test acceptance might be useful. In my opinion, at a bare minimum: should build successfully in arm-none-eabi 9.2.1 on windows and should not break the e2 Studio workflows (unless changing the workflows /is/ the change, I guess). I believe this is what @jamiefaye checks for at present.